File: 250911/가장 짧은 문자열/shortest-string.py
a=input()
b=input()
c=input()

arr = [a,b,c]

max_len = len(a)
min_len = len(a)

for i in range(1,3) :
    if max_len < len(arr[i]) :
        max_len = len(arr[i])
    if min_len > len(arr[i]) :
        min_len = len(arr[i])
    
print(max_len-min_len)

File: 250608/가장 짧은 문자열/shortest-string.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String str1 = sc.next();
        String str2 = sc.next();
        String str3 = sc.next();
        int maxlen = Math.max(str1.length(), Math.max(str2.length(), str3.length()));
        int minlen = Math.min(str1.length(), Math.min(str2.length(), str3.length()));


        System.out.println(maxlen-minlen);
    }
}
